setup-jest.ts

```diff
@@ -1 +1,5 @@
 import 'jest-preset-angular/setup-jest';
+
+Object.defineProperty(window, 'DragEvent', {
```

Comment:
Is dragEvent not supported in JsDom?  Can we try putting this code in the corresponding test file? That way, it will stay isolated. I kind of did something similar with ResizeObservers

Reply:
I tried but it didn't work. For `ResizeObserver` as well, we put in the `setup-jest.ts` file itself, right?